### DISCUSSION TITLE
RC turns off and rx PWM Ch3-(throttle) goes to 100% value

### DISCUSSION THREAD

**Discussion by Blastaar-BR** [ExpressLRS/ExpressLRS#1653] [Q&A] [answered]
Hi, 

I am using a Radiomaster T16s RC+Namimno 2.4 Flash Oled tx with ELRS 2.0,  the rx is  Matek ELRS-R24-P elrs 2.4ghz  5 ch PWM with ELRS 2.0 for a 5 channel plane, connected as:

- CH1 : Aileron servos (with Y cable)
- CH2: Elevator servo
- CH3: ESC/Motor
- CH4 - Rudder servo
- CH5 - not connected

I use a RC switch to override the Ch3 to  -100 when activated, as Motor Cut off safety fuction.

I turn on the radio,  connect the plane battery, the rx binds to the rc  and it flies perfectly.

After landing the throttle is in lowest position and the rc Motor Cut Off switch is activated to safely kill the motor.

In this situation, if I **turn off** the radio, the motor (CH3) immediately goes to a value that starts turning the motor in a considerable speed (very dangerous situation). Not sure how much in % to the max speed but it turns quite heavy.

Why is the ELRS rx doing that? What are the values the rx assume when a broken link happen? Are they "zero"? If yes, it might answer why the motor turns (In the radio the throttle stick is configured like  CH3 = -100 means motor in zero speed; CH3 = 0 means motor speed in 50%, CH3 = 100 means full throttle).

How to activate a failsafe functionality to ensure that in case of broken radio link it will go to a safe state (user should be able to define the values per channel)?

Any help would be appreciated!

**Comment by pkendall64** [comment, marked as answer]
You need to go to the web ui on the RX and set the failsafe values in the PWM section.

**Reply by Blastaar-BR** [reply]
Hi, thank you so much! For whatever reason the value in the ui failsafe CH3, was 1500. I tried to write 0, but it goes back to 988 (that seems to be the lowest value). I repeat the test and now when I turn off the rc, the ch3 goes off and motor stop turning! Thank you!

**Reply by pkendall64** [reply]
Yes, 988 is the minimum number of microseconds for CRSF signal. 2012 is the maximum and 1500 is the middle value.
These are the values that the opentx based handset uses as it's values as well.
